### ISSUE TITLE
Solus Doesn't Actually Work

### ISSUE THREAD

**Issue by devotoare** [cpriego/valet-linux#224]
Even after #127 was done and merged in the changes don't work for Solus.

For some reason many of @slave2anubis's changes were not part of the pull request and were not merged. All that happens as of now is `valet install` fails when trying to install FPM since Solus does not have FPM. If you get past that by modifying the cli//Valet/PhpFpm.php file then there's some issues with DNSMasq and you can't use the internet until you uninstall valet.

**Comment by jmarcher** [comment]
As I commented in the PR, I do not use Solus that is why I was appealing to Solus users to review that PR, feel free to do a PR with a fix.

**Comment by beganovich** [comment]
I'd like to handle this weekend this if someone above isn't working on it! =)

**Comment by devotoare** [comment]
I haven't had time yet @beganovich

**Comment by jmarcher** [comment]
@beganovich Did you had time to work on a PR for this?

**Comment by beganovich** [comment]
@jmarcher I planned to do some work this weekend. I moved from Solus for some time but I installed it back =)

If you want to take a look, just go ahead.

**Comment by jmarcher** [comment]
@beganovich I never used Solus before and right now I do not have time to install it and test valet over there.
